platforms/corda-ent/charts/notary-initial-registration/templates/job.yaml

```diff
@@ -57,20 +57,21 @@ spec:
                 KUBE_SA_TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
                 VAULT_TOKEN=$(curl -sS --request POST ${VAULT_ADDR}/v1/auth/${KUBERNETES_AUTH_PATH}/login -H "Content-Type: application/json" -d '{"role":"'"${VAULT_APP_ROLE}"'","jwt":"'"${KUBE_SA_TOKEN}"'"}' | jq -r 'if .errors then . else .auth.client_token end')
                 validateVaultResponse 'vault login token' "${VAULT_TOKEN}"
+                
                 echo "logged into vault"
 
                 # Creating dirs for storing the certificate
                 mkdir -p ${MOUNT_PATH}/trust-stores
 
                 # Fetching network-root-truststore certificates from vault
                 COUNTER=1
-                while [ "$COUNTER" -lt {{ $.Values.healthcheck.readinessthreshold }} ]
+                while [ "$COUNTER" -lt {{ $.Values.vault.retries }} ]
                 do
                     # get keystores from vault to see if certificates are created and put in vault
                     LOOKUP_SECRET_RESPONSE=$(curl -sS --header "X-Vault-Token: ${VAULT_TOKEN}" ${VAULT_ADDR}/v1/${CERTS_SECRET_PREFIX}/root/certs | jq -r 'if .errors then . else . end')
                     if echo ${LOOKUP_SECRET_RESPONSE} | grep "errors"
                     then
-                      sleep {{ $.Values.healthcheck.readinesscheckinterval }}
+                      sleep {{ $.Values.vault.retryInterval }}

```

Comment:
To keep this consistent with other charts, use `vault.sleepTimeAfterError`

Reply:
retryInterval makes more sense to me as we want it is intended to retry in case of vault item not there